content/developer/reference/backend/testing.rst

```diff
@@ -819,35 +840,39 @@ driven explicitly by the user) it's more complicated when running
 "test" tours, or when running tours through the test suite. In that
 case there are two main tricks:
 
-- Have a step with a ``run() { debugger; }`` action.
+- Have a step property ``break: true,`` in debug mode (debug=True).
+
+  This adds a debugger at the start of the step and then allows you
+  to put breakpoints in the desired locations.
+
+**Advantages**
+  - very simple
+  - the tour restarts as soon as you resume execution
+**Drawbacks**
+  - page interaction is limited as all javascript is blocked
+  - Have a step property ``pause: true,`` in debug mode (debug=True).
+
+  The tour will stop at the end of the step. This allows inspecting
+  and interacting with the page until the developer is ready to
+  resume by typing **play();** in console.
+
+**Advantages**
+  - allows interacting with the page
+  - no useless (for this situation) debugger UI
+  - Have a step with a ``run() { debugger; }`` action.
```

Comment:
Again, I'd remove the "Have".
It seems the indention is broken again as this should be a new mini section, but maybe it just looks broken on github.

Reply:
I'll check